MinimedKit/PumpManager/MinimedPumpManager.swift

```diff
@@ -364,6 +364,7 @@ extension MinimedPumpManager {
                     date: date,
                     quantity: HKQuantity(unit: .milligramsPerDeciliter, doubleValue: Double(glucose)),
                     isDisplayOnly: false,
+                    wasUserEntered: false,
```

Comment:
[nit] How do you know this wasn't used entered? Doesn't this pump allow the user to enter BG values?